knowledge-base/generate-doc-template-and-populate-with-collection-data-mail-merge.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Generating a Word Document Template with Data Using MailMerge in RadWordsProcessing
```

Comment:
Maybe add the following clarification throughout the whole KB - generating a Word document template *and populating it* with data/collection of data using the mail merge functionality

This can prevent misunderstandings because some might interpret this as - the final result is a template that is somehow generated with collection of data. For example the data will be used to generate the placeholders. 

However, the final result will not be a template but a normal document (from a populated template).